core/templates/domain/exploration/ExplorationMetadataObjectFactory.ts

```diff
@@ -40,6 +40,8 @@ export interface ExplorationMetadataBackendDict {
   param_changes: ParamChangeBackendDict[];
   auto_tts_enabled: boolean;
   edits_allowed: boolean;
+  has_viewed_lesson_info_modal_once: boolean;
```

Comment:
Just to check, why do changes need to be made here if the backend API is not being modified? Also the new fields do not seem like "exploration metadata".

Ditto below. I don't think the modelling is correct here.